apps/hosts/views.py

```diff
@@ -189,3 +189,27 @@ def create_challenge_host_team(request):
         challenge_host.save()
         return Response(response_data, status=status.HTTP_201_CREATED)
     return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+
+
+@throttle_classes([UserRateThrottle, ])
+@api_view(['DELETE', ])
+@permission_classes((permissions.IsAuthenticated, HasVerifiedEmail))
+@authentication_classes((ExpiringTokenAuthentication, ))
+def delete_challenge_host_from_team_by_self(request, challenge_host_team_pk):
+    """
+    A user can remove himself from the challenge host team.
+    """
+    try:
+        challenge_host = ChallengeHostTeam.objects.get(pk=challenge_host_team_pk)
```

Comment:
If I change the varibale name to `challenge_host_team` then the TravisCI build fails, as it is only used to check whether a team exists or not but `challenge_host_team` isn't used anywhere in the function.